snap/local/helper-go/install.go

```diff
@@ -283,8 +283,15 @@ func install() {
 
 	var err error
 
-	if err = installConfFiles(); err != nil {
-		log.Fatalf("Error installing config files: %v", err)
+	// Install default config files only if no config provider is connected
+	isConnected, err := snapctl.IsConnected("edgex-config").Run()
+	if err != nil {
+		log.Fatalf("Error checking interface connection: %s", err)
+	}
```

Comment:
it was intended to be here in preparation for a possible config provider plug in the future. Do you think it would be better to remove it?

Reply:
Yes, please remove. Snapctl throws an error when the interface doesn't exist.

Reply:
Inside the snap:
```
$ snapctl is-connected my-non-exited-interface
error: error running snapctl: snap "my-snap" has no plug or slot named "my-non-exited-interface"
$ echo $?
1
```
There is no fatal error because the library has a bug. We design and build on top of that.